tests/loginPage.ts

```diff
@@ -0,0 +1,29 @@
+import { Credential } from "./creditintial";
+import { Locator, Page } from "@playwright/test";
+
+export default class loginPage {
+  private pageDomain: Page;
+
+  usernameTextBox: Locator;
+  passwordTextBox: Locator;
+  loginButton: Locator;
+  invalidCredential: Locator;
+  credentialList: Credential[] = [
```

Comment:
why did you create a class for the credentials?

Reply:
this class used to store each credentials that is used for testing.